sdk/core/Microsoft.Azure.Core.NewtonsoftJson/README.md

```diff
@@ -1,11 +1,13 @@
 # Newtonsoft.Json support for Azure Core shared client library for .NET
 
-The [Azure.Core package][azure_core_package] contains types shared by all Azure SDK client libraries.
-This library contains converters dependent on the [Newtonsoft.Json package][newtonsoft_json_package] for use with Azure.Core.
+The [Azure.Core package][azure_core_package] contains types shared by the latest Azure SDK client libraries. This `Newtonsoft.Json` compatibility library:
 
-## Getting started
+- Contains converters dependent upon the [Newtonsoft.Json][newtonsoft_json_package] package.
+- Enables `Newtonsoft.Json` usage for serializing and deserializing model types from the following client libraries:
+  - [Azure.DigitalTwins.Core][azure_digital_twins_core_package]
+  - [Azure.Search.Documents][azure_search_documents_package]
```

Comment:
The model types are the ones that are serialized (so customer data), not types from the Azure libraries.

Reply:
If I change the word "from" to "with", does that work for you?

Reply:
yes, I think so.

Reply:
+1. You could may change "model types *from* the following" to something like "model types for use with the following".

Reply:
I think it's still rather vague and we'll get questions about it not working as expected. For example, recently-ish people were trying to serialize Key Vault SDK models and didn't come out like they expected (we don't support it). If you're changing just "from" to "with", I think you need to say "customer model types".

Reply:
@heaths Please review the latest changes.